addons/website/views/snippets/s_banner.xml

```diff
@@ -2,19 +2,41 @@
 <odoo>
 
 <template id="s_banner" name="Banner">
-    <section class="s_banner parallax s_parallax_is_fixed pt96 pb96" data-scroll-background-ratio="1">
+    <section class="s_banner parallax s_parallax_is_fixed o_half_screen_height" data-scroll-background-ratio="1">
         <span class="s_parallax_bg oe_img_bg" style="background-image: url('/web/image/website.s_banner_default_image'); background-position: 50% 0;"/>
         <div class="container">
-            <div class="row s_nb_column_fixed">
-                <div class="col-lg-7 bg-white jumbotron rounded pt32 pb32" data-name="Box">
+            <div class="row no-gutters s_nb_column_fixed">
+                <div class="s_banner_box col-lg-7 bg-white p-5 lead" data-name="Box">
```

Comment:
@Cocographique Can you move this one in another PR, there needs to be lot more discussion about this. Using no-gutters is not a good solution to me. And if it is, it should probably a generic solution for all snippets, no just banner.

Reply:
**PR** [46922](https://github.com/odoo/odoo/pull/46922)